hastexo/common.py

```diff
@@ -148,6 +148,9 @@
 
 DEFAULT_SETTINGS = {
     "terminal_url": "/hastexo-xblock/",
+    "terminal_color_scheme": "white-black",
+    "terminal_font": "monospace",
```

Comment:
Perhaps call this `terminal_font_name` to maintain parallelism with Guacamole options? (This would also require an update to the commit message.)

Reply:
Good point, will do that.